src/main/java/edu/ksu/canvas/attendance/submitter/CanvasAssignmentAssistant.java

```diff
@@ -25,10 +25,10 @@
 public class CanvasAssignmentAssistant {
 
     private static final Logger LOG = Logger.getLogger(CanvasAssignmentAssistant.class);
-    private static final String ASSIGNMENT_DESCRIPTION ="This assignment is based on attendance recorded in the K-State " +
-                                                        "Attendance tool. Do not make changes to attendance here; changes need to be made in the K-State Attendance tool. " +
-                                                        "Information about the number of classes in which a student was present, tardy, absent, and excused is " +
-                                                        "identified in the comments area of the student's grade.";
+    private static final String ASSIGNMENT_DESCRIPTION ="This is a single Assignment that reflects the attendance records with an Assignment Score. " +
+                                                        "Any changes to the Attendance Assignment Name, Attendance Weights, or Total Points will be updated in the single assignment when you select Push Assignment to Canvas. " +
+                                                        "<html><strong>DO NOT edit the Attendance Assignment Name or Total Points in the Assignment tool.</strong></html> " +
```

Comment:
Do you really need the `html` tag around the  `strong` tag?  Shouldn't this just be the `strong` tag?  Also, the example text only has "Do not" in bold. This has the entire sentence in bold.